packages/debug-ui/src/code/indexEntry.ts

```diff
@@ -0,0 +1,26 @@
+import { Civil } from "@joincivil/core";
+import * as marked from "marked";
+import BN from "bignumber.js";
+
+import { setIndexListeners } from "./listeners";
+
+// Metamask is injected after full load
+window.addEventListener("load", async () => {
+  setIndexListeners();
+  const civil = new Civil();
```

Comment:
```new Civil({debug: true});``` it's a debug-ui after all

Reply:
Frankly there should be some util.ts with `createCivilInstance()`, because we're creating it in many places